src/docs/guides/django.md

```diff
@@ -0,0 +1,254 @@
+        - `PGDATABASE`: Set the value to `${{Postgres.PGDATABASE}}` (this references the Postgres database name). Learn more about [referencing service variables](/guides/variables#referencing-another-services-variable).  
+        - `PGUSER`: Set the value to `${{Postgres.PGUSER}}`
+        - `PGPASSWORD`: Set the value to `${{Postgres.PGPASSWORD}}`
+        - `PGHOST`: Set the value to `${{Postgres.PGHOST}}`
+        - `PGPORT`: Set the value to `${{Postgres.PGPORT}}`
+    - Use the **Raw Editor** to add any other required environment variables in one go.
+6. **Redeploy the App Service**:
+    - Click **Deploy** on the app service on the Railway dashboard to apply your changes.
+7. **Verify the Deployment**:
+    - Once the deployment completes, go to **View logs** to check if the server is running successfully.
+8. **Set Up a Public URL**:
+    - Navigate to the **Networking** section under the [Settings](/overview/the-basics#service-settings) tab of your new service.
+    - Click [Generate Domain](/guides/public-networking#railway-provided-domain) to create a public URL for your app.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1729121823/docs/quick-start/django_app.png"
+alt="screenshot of the deployed Django project"
+layout="responsive"
+width={2783} height={2135} quality={100} />
+
+
+## Deploy from a GitHub Repo
+
+To deploy the Django app to Railway, start by pushing the app to a GitHub repo. Once that’s set up, follow the steps below to complete the deployment process.
+
+1. **Create a New Project on Railway**:
+    - Go to <a href="https://railway.app/new" target="_blank">Railway</a> to create a new project.
+2. **Deploy from GitHub**: 
+    - Select **Deploy from GitHub repo** and choose your repository.
+        - If your Railway account isn’t linked to GitHub yet, you’ll be prompted to do so.
+3. **Add Environment Variables**:
+    - Click **Add Variables** and configure all the necessary environment variables for your app.
+        - `PGDATABASE`: Set the value to `${{Postgres.PGDATABASE}}` (this references the Postgres database name). Learn more about [referencing service variables](/guides/variables#referencing-another-services-variable).  
+        - `PGUSER`: Set the value to `${{Postgres.PGUSER}}`
+        - `PGPASSWORD`: Set the value to `${{Postgres.PGPASSWORD}}`
+        - `PGHOST`: Set the value to `${{Postgres.PGHOST}}`
+        - `PGPORT`: Set the value to `${{Postgres.PGPORT}}`
+    **Note:** We don't have the Postgres Database service yet. We'll add that soon.
+4. **Deploy the App**: 
+    - Click **Deploy** to start the deployment process.
+    - Once the deployed, a Railway [service](/guides/services) will be created for your app, but it won’t be publicly accessible by default.
+    **Note:** You'll encounter an error about the PGDATABASE environment not set. Don't worry, we'll fix that soon.
+5. **Add a Database Service**:
```

Comment:
i know the add environment variables mode is designed to add variables, but is there anything stopping you from also adding a database in that time too?

Reply:
Actually that's a good idea, because the database will still get provisioned if I add it.  I'll move the step to add the database up

Reply:
Updated!